_posts/2023-06-11-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,195 @@
+
+Atlas is built with VueJS and designed to be easily navigatable, displaying all image details clearly and intuitively. Users can read about the build steps, size, layers, and other details for creating a variant of Vanilla OS.
+
+Currently, the platform showcases the Pico, Core, Desktop and Dev images, but this list might expand in the future to include new compliant images.
+
+You can test the platform by visiting [images.vanillaos.org](https://images.vanillaos.org/).
+
+*Atlas was the Titan of Greek mythology condemned to hold up the heavens or sky for eternity.
+
+## New Wallpaper
+
+Like every Linux distribution that sees a new major update, Vanilla OS Orchid has received a new official wallpaper. [Kramo](https://github.com/kra-mo) has been working hard on this. He has made a wallpaper highlighting the new light that Orchid brings to Vanilla OS.
+
+![Orchid Wallpaper, light variant: A 2D image of vanilla orchid over a sky-based wallpaper](/assets/uploads/orchid-light.webp)
+
+The new wallpaper comes in two colors, light and dark, and it represents a vanilla orchid in a pure environment, which embodies the pure essence of Vanilla OS and the desire to provide a simple and relaxing user experience.
+
+![Orchid Wallpaper, dark variant: A 2D image of vanilla orchid over a plant-based wallpaper](/assets/uploads/orchid-dark.webp)
+
+## Other news
+
+There are many new features we have worked on, and we cannot cover them all in detail in a single article for various reasons. However, we can briefly mention some of them.
+
+As announced in the previous article, we have been working on changes to support OEM in Vanilla OS. This includes moving user creation and other installer steps to [First Setup](https://github.com/Vanilla-OS/first-setup) and improving the user experience of both the Installer and First Setup.
+
+We are also developing an intelligent graphics driver manager based on [Ikaros](https://github.com/Vanilla-OS/ikaros) that suggests driver installations whenever needed, guiding the user without interrupting their workflow.
+
+![Drivers Utility](/assets/uploads/drivers-utility.webp)
+
+Among the various new features we are working on for the installer is a new Recovery mode with easy access to all recovery tools like the terminal, ABRoot recovery, the browser, and more. Additionally, we are working on a recovery partition that allows graphical restoration of one's Vanilla OS installation if anything gets compromised due to external causes.
+
+![Recovery utility](/assets/uploads/recovery-utility.webp)
+
+---
+
+The development of Vanilla OS 2.0 is progressing rapidly, and more news will follow soon!
+
+For latest updates, checkout the [roadmap](https://vanillaos.org/roadmap) and the [project tracker](https://github.com/orgs/Vanilla-OS/projects/4).
```

Comment:
```suggestion
For the latest updates, checkout the [roadmap](https://vanillaos.org/roadmap) and the [project tracker](https://github.com/orgs/Vanilla-OS/projects/4).
```

Reply:
I think "checkout" should be "check out"

Reply:
Added, thanks